l5kit/l5kit/evaluation/metrics.py

```diff
@@ -1,46 +1,299 @@
 import numpy as np
 
 
-def neg_multi_log_likelihood(
-    gt: np.ndarray, pred: np.ndarray, confidences: np.ndarray, avails: np.ndarray
-) -> np.ndarray:
+def _assert_shapes(ground_truth: np.ndarray, pred: np.ndarray, confidences: np.ndarray, avails: np.ndarray) -> None:
```

Comment:
Do we enforce some number of modes for the competition? Might be worthwhile checking that in the computation if yes. Otherwise it seems any number of modes is accepted. Is there perhaps a maximum we should enforce at least?

And I assume that we expect the same number of modes for all predictions, i.e., we can't predict 3 for one situation and 5 for another.